src/gorest.py

```diff
@@ -0,0 +1,17 @@
+import requests
+import json
+
+api_url = "https://gorest.co.in/public/v2/users"
+user_details_api_url = "https://gorest.co.in/public/v2/users/"
+
+def fetch_user_details():
+    user_details_list = []
+    user_list = json.loads((requests.get(url=api_url)).text)
+    for user in user_list:
+        user_id = user["id"]
+        user_details = json.loads((requests.get(url = user_details_api_url+str(user_id))).text)
+        user_details_list.append(user_details)
```

Comment:
there is no need to use json.loads().text instead you can just use json.loads().